src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -481,7 +481,79 @@ let ``Supports default custom simulator`` () =
     given ["--simulator"; typeof<QuantumSimulator>.FullName; "--use-h"; "false"] |> fails
 
 
-// TODO: Add tests for the "submit" command.
+// Azure Quantum Submission
+
+/// Standard command-line arguments for the "submit" command using the "nothing" target.
+let private standardSubmitArgs =
+    ["submit"
+     "--target"
+     "nothing"
+     "--storage"
+     "myStorage"
+     "--subscription"
+     "mySubscription"
+     "--resource-group"
+     "myResourceGroup"
+     "--workspace"
+     "myWorkspace"]
+
+[<Fact>]
+let ``Submit can submit a job`` () =
```

Comment:
It might be worth it to add a test case in which an unexpected exception occurs when calling the submit method. Catching the exception and showing an output that lets the user know something unexpected happened would be valuable.

Reply:
I think @bettinaheim and I were wanting to wait until next release to improve the exception handling. However, I can add a test where an exception is thrown and check that the first line or so of the exception stack trace is correct. Would that work for now?

Reply:
I see that you improved the exception handling in #218, so I can add tests for those instead now.

Reply:
I'll include this in the next tests PR (see comment at the bottom).